menas/src/main/scala/za/co/absa/enceladus/menas/auth/kerberos/MenasKerberosLdapContextSource.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas.auth.kerberos
+
+import java.security.PrivilegedAction
+import java.util.Hashtable
+
+import org.springframework.security.ldap.DefaultSpringSecurityContextSource
+
+import javax.naming.Context
+import javax.naming.directory.DirContext
+import javax.security.auth.Subject
+
+class MenasKerberosLdapContextSource(url: String, subject: Subject) extends DefaultSpringSecurityContextSource(url) {
+  override def getDirContextInstance(environment: Hashtable[String, Object]): DirContext = {
```

Comment:
My IDE thinks that this `Hastable` reference must be prefixed. We could either do that or "alias" the import like 
```
import java.util.{Hashtable => uHashTable}
```

Reply:
Gonna prefix inline.